services/shared/src/main/java/com/wedriveu/services/shared/rabbitmq/client/RabbitMQFactory.java

```diff
@@ -0,0 +1,29 @@
+package com.wedriveu.services.shared.rabbitmq.client;
+
+import com.wedriveu.services.shared.utilities.Constants;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.rabbitmq.RabbitMQClient;
+
+/**
+ * Created by Marco on 28/07/2017.
+ */
+public class RabbitMQFactory {
+
+    private static final String PASSWORD = "password";
+    private static final String HOST = "host";
+    private static final String PORT = "port";
+
+    public static RabbitMQClient createClient(Vertx vertx) {
+        return setupRabbitMQ(vertx);
+    }
+
+    private static RabbitMQClient setupRabbitMQ(Vertx vertx) {
```

Comment:
Why the body of this method can't be put directly inside the `createClient` one?